app/assets/javascripts/components/common/multi_select_field.jsx

```diff
@@ -47,4 +47,4 @@ const MultiSelectField = createClass({
   }
 });
 
-module.exports = MultiSelectField;
+export default MultiSelectField;
```

Comment:
The previous line was causing a warning. Hence made this change to remove the warning and to keep things more in line with how it is elsewhere.